src/db/migrations/20200220144524-create-request.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         onUpdate: 'CASCADE',
         references: {
           model: 'Users',
-          key: 'userId',
+          key: 'id',
```

Comment:
You may not want to edit an existing migration. My Database schema was not updated because this migration was run before. Create a new migration file and make the changes you wish to make.

Reply:
you can run the `npm run createTables` in your terminal